### PR TITLE
Task/WG 370 open topo UI improvements

### PR DESCRIPTION
**Pull request by sophia-massie**
## PR Status: ##

* [X] Ready.
* [ ] Work in Progress.
* [ ] Hold.

## Related Jira tickets: ##

* [WG-370](https://tacc-main.atlassian.net/browse/WG-370)

## Testing Steps: ##
1. Go to https://designsafe.dev/recon-portal/
2. Click 'Show filter options' above the map
3. Click View External Dataset Filters
4. Make sure listings and event markers appear/hide as is appropriate 

## UI Photos:
Screenshot 2024-08-27 at 7 16 50 PM
Screenshot 2024-08-27 at 7 16 35 PM

## Notes: ##
External Data 2 will only mimic the open topo filter. Implementation will need to come later - just put this placeholder for CMD to work with and keep in mind. Can remove before merging.

### CODE REVIEW

File: .nx/cache/18.3.2-nx.darwin-x64.node
Comment (nathanfranklin) [inline .nx/cache/18.3.2-nx.darwin-x64.node:1]: I see this nx file made its way in.  should we drop and then hotfix something to the .gitignore to ingore the .nx folder?

**Review by nathanfranklin — CHANGES_REQUESTED** [review]
LGTM. looks great! 

just one comment on the extra file.

**Review by shayanaijaz — APPROVED** [review]
LGTM!

File: .nx/cache/18.3.2-nx.darwin-x64.node
Comment (sophia-massie) [reply .nx/cache/18.3.2-nx.darwin-x64.node:1]: Ah sorry! This wasn't even registering locally as an added file. It's now removed

**Review by nathanfranklin — APPROVED** [review]
LGTM 👍

**Pull request merged**